_includes/head.html

```diff
@@ -31,4 +31,5 @@
   </style>
 <![endif]-->
 
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<meta http-equiv="cleartype" content="on">
+<script async defer src="{{ '/assets/js/main.min.js' | absolute_url }}"></script>
```

Comment:
Guess I should have kept scrolling down the page because the W3 said exactly what you did 😛 

> The defer attribute may be specified even if the async attribute is specified, to cause legacy Web browsers that only support defer (and not async) to fall back to the defer behavior instead of the synchronous blocking behavior that is the default.